.rubocop.yml

```diff
@@ -69,6 +70,7 @@ Layout/LineLength:
 
 Metrics/BlockLength:
   Exclude:
+    - app/admin/admin_users.rb
```

Comment:
Propose to excluded `register ` methods 

Look at the [doc](https://docs.rubocop.org/rubocop/cops_metrics.html#metricsblocklength)